Display.h

```diff
@@ -121,26 +125,32 @@ class Display {
   Display(LiquidCrystal* lcd, const float& trigger_threshold):
       lcd_(lcd),
       trigger_threshold_(trigger_threshold),
-      animation_(1000, 0.5) {
-    elements_[HEADER]       = Element{0, 0, 20};
-    elements_[VOLUME]       = Element{1, 0, 11, "TV"};
-    elements_[BPM]          = Element{2, 0, 11, "RR"};
-    elements_[IE_RATIO]     = Element{3, 0, 11, "IE"};
-    elements_[AC_TRIGGER]   = Element{0, 0, 11, "AC"};
-    elements_[PRES_LABEL]   = Element{0, 11, 9};
-    elements_[PEAK_PRES]    = Element{1, 11, 9, "peak"};
-    elements_[PLATEAU_PRES] = Element{2, 11, 9, "plat"};
-    elements_[PEEP_PRES]    = Element{3, 11, 9, "PEEP"};
+      animation_(1000, 0.5),
+      animationfooter_(1000, 0.5),
+      snoozecountdown_(0) {
+    elements_[HEADER]       = Element{0, 0, kHeaderLength};
+    elements_[VOLUME]       = Element{1, 0, 4, "TV"};
+    elements_[BPM]          = Element{1, 4, 3, "RR"};
+    elements_[IE_RATIO]     = Element{1, 7, 5, "IE"};
+    elements_[AC_TRIGGER]   = Element{1, 12, 5, "Ptr"};
+    elements_[PEAK_PRES]    = Element{1, 17, 3, "peak"};
+    elements_[PLATEAU_PRES] = Element{2, 17, 3, "plat"};
+    elements_[PEEP_PRES]    = Element{3, 17, 3, "PEEP"};
+    elements_[SNOOZE]       = Element{3, 1, 3};
+    elements_[FOOTER]       = Element{3, 7, kFooterLength};
```

Comment:
Documentation should reflect these changes

Reply:
We plan to document the new display layout on: https://emergency-vent.mit.edu/.